src/fmu/sumo/sim2sumo/sim2sumo.py

```diff
@@ -167,11 +153,12 @@ def export_results(
     return exp_path
 
 
-def read_config(config):
+def read_config(config, **kwargs):
```

Comment:
can this be two separate values instead of `**kwargs`?
e.g. `datafile=None` and `datatype=None`